### ISSUE TITLE
Host-Only-Adapter has wrong IP and subnetmask 

### ISSUE THREAD

**Issue by athiele** [hashicorp/vagrant#1014]
When starting a box with host-only-networking, sometimes the created virtual network adapter in VirtualBox has a different IP and subnetmask than specified. That causes the problem, that if I start an other box of our Vagrant environment, VirtualBox registers a "Virtualbox Host Only Adapter #2", #3, etc. This prevents the different VMs from communicating with each other, because the first VMs network card is "Virtualbox Host Only Adapter", and the second VMs adapter is "Virtualbox Host Only Adapter #2".

This problem occurs sporadically on different hosts with Windows 7. After rebooting the host machine, the error usually does not occur anymore.

The Vagrantfile we use:

```
Vagrant::Config.run do |config|
  config.vm.define :management do |man_config|

    [...]
    man_config.vm.boot_mode = :headless
    man_config.vm.network :hostonly, "192.168.0.10", :netmask => "255.255.255.0"
    man_config.vm.customize ["modifyvm", :id, "--memory", 2200]
    man_config.vm.customize ["modifyvm", :id, "--cpus", 2]
    [...]
  end
[..] 
end

```

In the Virtualbox Console, "VirtualBox Host-Only Ethernet Adapter" has the following properties:

```
IPv4 address: 169.254.130.104
IPv4 netmask: 255.255.0.0

```

We found this workaround, to force the VMs to use one network adapter, even if it has the wrong ip data :
`man_config.vm.customize ["modifyvm", :id, "--nic2", "hostonly", "--cableconnected2", "on", "--hostonlyadapter2", "VirtualBox Host-Only Ethernet Adapter"]`

This workaround is very helpful but it seems to be not a clean solution for permanent usage. 

Software in Use:
Windows 7 64bit
Vagrant 1.0.3
VirtualBox 4.1.16

Thanks in advance and kind regards
Aimo

**Comment by rogersanctus** [comment]
Well, I had the same problem on windows 8.1, but I just had to rerun the vagrant up command again and it worked xD.

**Comment by berpcor** [comment]
The same problem. I upgraded Virtualbox and Vagrant to the latest versions, and thus I created problems for me. Downgrade doesn't help. Every time I do vagrant up, vagrant creates a new host-only adapter with the wrong settings.

```
  config.vm.network :private_network, ip: 192.168.56.56
  config.vm.network :forwarded_port, guest: 80, host: 8000
  config.vm.network :forwarded_port, guest: 9200, host: 9200
```
IP and mask for a new adapter (if it doesn't exists) should be 192.168.56.1 and 255.255.255.0 but like it is said above it is (169.254.x.x).
